reference-implementation/to-upstream-wpts/writable-streams/close.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 if (self.importScripts) {
-  self.importScripts('/resources/testharness.js');
+  self.importScripts('/resources/testharness.js', '..resources/test-utils.js');
```

Comment:
Hah, I forgot you could do that. Maybe a separate line though for consistency with the rest of the imports?